troi/patches/ab_similar_recordings.py

```diff
@@ -6,13 +8,26 @@
 from troi.acousticbrainz import annoy
 
 
+@click.group()
+def cli():
+    pass
+
+
 class ABSimilarRecordingsPatch(troi.patch.Patch):
 
     @staticmethod
-    def inputs():
-        return [
-            {"type": str, "name": "recording_id", "desc": "MusicBrainz recording id", "optional": False},
-            {"type": str, "name": "similarity_type", "desc": "Type of acousticbrainz similarity", "optional": False}]
+    @cli.command(no_args_is_help=True)
+    @click.argument('recording_id')
+    @click.argument('similarity_type')
+    def parse_args(**kwargs):
+        """
+        Find acoustically similar recordings from AcousticBrainz.
+
+        \b
```

Comment:
what magic is this?

Reply:
click magically reflows paragraphs based on the width of your terminal. This forces click to treat the following as verbatim lines